### PR TITLE
Added Absolute/Relative Copy Path.

### PR DESCRIPTION
**Pull request by Jerrody**

### CODE REVIEW

File: lapce-ui/Cargo.toml [hunk 1]
```diff
@@ -42,10 +42,14 @@ xi-unicode = "0.3.0"
...
-open = { version = "3.0.2", features = [] }
+open = "3.0.2"
 
 # lapce deps
-druid = { git = "https://github.com/lapce/druid", branch = "shell_opengl", features = [ "svg", "im", "serde", ] }
+druid = { git = "https://github.com/lapce/druid", branch = "shell_opengl", features = [
+    "svg",
+    "im",
+    "serde",
+] }
```
Comment (panekj) [inline lapce-ui/Cargo.toml:52]: Please remove unnecessary reformatting

**Review by panekj — CHANGES_REQUESTED** [review]

File: lapce-ui/Cargo.toml (hunk 1 above)
Comment (Jerrody) [reply lapce-ui/Cargo.toml:52]: > Please remove unnecessary reformatting
Alright.

File: lapce-ui/src/explorer.rs [hunk 2]
```diff
@@ -686,6 +690,29 @@ impl Widget<LapceTabData> for FileExplorerFileList {
                             menu = menu.entry(item);
                         }
 
+                        menu = menu.separator();
+                        let path_to_file = node.path_buf.clone();
+                        let item = druid::MenuItem::new("Copy Absolute Path")
```
Comment (bugadani) [inline lapce-ui/src/explorer.rs:695]: VSCode calls this "Copy Path" and I also think "Absolute" is not really needed here.

Comment (Jerrody) [reply lapce-ui/src/explorer.rs:695]: Sometimes it's necessary. I think that is good when the user has an variants and can choose.

Comment (panekj) [reply lapce-ui/src/explorer.rs:695]: It should be:
`Copy path`
`Copy relative path`

Comment (bugadani) [reply lapce-ui/src/explorer.rs:695]: Sorry I meant the word isn't necessary, so the options should be "Copy Path" and "Copy Relative Path". This is what VSCode calls these, too.

Comment (Jerrody) [reply lapce-ui/src/explorer.rs:695]: > Sorry I meant the word isn't necessary, so the options should be "Copy Path" and "Copy Relative Path". This is what VSCode calls these, too.

Oh, okay. I missunderstood, well, I do not insist in this case, I just like to be meticulous.
Anyway, if you think that would be better to remove this word, welp.

Comment (panekj) [reply lapce-ui/src/explorer.rs]: I think we just meant the UI text, not internal code

Comment (Jerrody) [reply lapce-ui/src/explorer.rs]: > I think we just meant the UI text, not internal code

Then the uniformity of the code is lost.

**Pull request merged**